docs/tutorials/deploy_sample_app.md

```diff
@@ -287,291 +125,120 @@ Practical notes about cycles:
+  /favicon.ico 1/1 (15406 bytes) sha 4e8d31b50ffb59695389d94e393d299c5693405a12f6ccd08c31bcf9b58db2d4 (with 7 headers)
+  /index.html (gzip) 1/1 (350 bytes) sha 16289744897bd78f5df24924dac6972c19e0bb56f5ddcf695de65656b942d769 (with 7 headers)
+  /logo2.svg 1/1 (15139 bytes) sha 037eb7ae523403daa588cf4f47a34c56a3f5de08a5a2dd2364839e45f14f4b8b (with 7 headers)
+  /index.js.LICENSE.txt (gzip) 1/1 (273 bytes) sha db89b3ccdfe399f8ef3135c0b076326a0ae9e1c96409f79f8e686031537c572c (with 7 headers)
+  /index.js (gzip) 1/1 (88325 bytes) sha 37809370db58979a0bd92a68eb403eb06f9314748862376e2b4ac2d6d171e631 (with 7 headers)
+  /main.css (gzip) 1/1 (299 bytes) sha b4879e7ba34e68b2965d626e48d772ce615e4f6b78b69cc8f2f91127ed18b850 (with 7 headers)
+  /index.html 1/1 (539 bytes) sha 053f9dc1283c64d114d43cbf03b0b0062afae08a04a5044ae58dbc68f4a1f93f (with 7 headers)
+  /index.js 1/1 (246603 bytes) sha 0de3c2e257ca0ac85423b3e4c7dc0a3aeb8906b73e3b72e47924ed6e80247406 (with 7 headers)
+Committing batch.
 Deployed canisters.
 URLs:
-  Frontend:
-    hello_frontend: https://5h5yf-eiaaa-aaaaa-qaada-cai.icp0.io/
-  Candid:
-    hello_backend: https://a4gq6-oaaaa-aaaab-qaa4q-cai.raw.icp0.io/?id=5o6tz-saaaa-aaaaa-qaacq-cai
+  Frontend canister via browser
+    hello_frontend: https://jqylk-byaaa-aaaal-qbymq-cai.icp0.io/
+  Backend canister via Candid interface:
+    hello_backend: https://a4gq6-oaaaa-aaaab-qaa4q-cai.raw.icp0.io/?id=jxzn6-maaaa-aaaal-qbyma-cai
 ```
 
-Note the bottom of the message which returns the URL where you can see your canister’s frontend deployed on-chain: <https://5h5yf-eiaaa-aaaaa-qaada-cai.icp0.io/>
-
-In the example above, we created a `hello` dapp that is composed of:
-
-1.  `hello_backend` canister `5o6tz-saaaa-aaaaa-qaacq-cai` which contains the backend logic.
-
-2.  `hello_frontend` canister `5h5yf-eiaaa-aaaaa-qaada-cai` which contains the frontend assets (e.g. HTML, JS, CSS).
-
-### See your dapp live on-chain via a browser
-
-Navigate to and enter a name: <https://5h5yf-eiaaa-aaaaa-qaada-cai.icp0.io/>
-
-Before your dapp loads, your browser will quickly show a message that reads: Installing "Internet Computer Validating Service Worker". This [service worker](https://developer.chrome.com/docs/workbox/service-worker-overview/) comes from the IC and it is used to make sure the web app the user sees is the correct, untampered frontend. Once loaded, your browser will cache the service worker and your web app will load much quicker.
-
-![Hello](_attachments/service-worker.png)
-
-After loading the service worker, your dapp will load:
+Note the bottom of the message which returns the URL where you can see your canister’s frontend deployed on-chain: <https://jqylk-byaaa-aaaal-qbymq-cai.icp0.io/>
 
-![Hello](_attachments/frontend-result.png)
+### 🎉 Success, you are done! 🎉
 
-### Testing the on-chain dapp via the command line
+### You have successfully deployed your dapp on-chain
```

Comment:
Should both of these be headings?

Reply:
not really, i just wanted them to pop. they were originally bold, but i thought it was not big enough. you caught me :)

Reply:
fixed